Carmu/Sources/Controllers/CrewMake/Crew/BoardingPointSelectViewController.swift

```diff
@@ -66,13 +107,38 @@ extension BoardingPointSelectViewController {
     }
 }
 
+// MARK: - custom Method
+extension BoardingPointSelectViewController {
+
+    /**
+     경유지 선택 화면에서 선택된 버튼의 인덱스를 String으로 변환하는 메서드
+     */
+    private func setUserToPoint() -> String {
+        switch selectedPoint {
+        case 0:
+            return "startingPoint"
+        case 1:
+            return "stopover1"
+        case 2:
+            return "stopover2"
+        default:
+            return "stopover3"
+        }
+    }
```

Comment:
열거형을 활용해서 이렇게도 써볼 수 있을 것 같습니다. 선택한 인덱스 값으로 `SelectedPoint.allCases`에 접근해서 `rawValue`를 뽑아내는... 참고만 해보시죠
```suggestion
    enum SelectedPoint: String, CaseIterable {
        case startingPoint = "startingPoint"
        case stopover1 = "stopover1"
        case stopover2 = "stopover2"
        case stopover3 = "stopover3"
        }
    }
```